falkordb-cluster/cluster-entrypoint.sh

```diff
@@ -276,6 +385,7 @@ run_node() {
     echo "tls-ca-cert-file $ROOT_CA_PATH" >>$NODE_CONF_FILE
     echo "tls-cluster yes" >>$NODE_CONF_FILE
     echo "tls-auth-clients no" >>$NODE_CONF_FILE
+    echo "tls-replication yes" >>$NODE_CONF_FILE
```

Comment:
Are you sure `tls-replication` in cluster is required? I see you removed the `tls-cluster` config from the node.conf. Is that expected?